serverless/src/cloudfrontToCloudwatch/handler.js

```diff
@@ -23,7 +23,7 @@ const cloudfrontToCloudwatch = async (event) => {
 
   console.log(`Processing ${s3Records.length} files(s)`)
 
-  s3 = new AWS.S3()
+  s3Client = new S3Client({ Region: 'us-east-1' })
```

Comment:
So in https://github.com/nasa/cmr-graphql/blob/main/src/cloudfrontToCloudwatch/handler.js
we don't pass a region I think we should keep that out of the SC
```suggestion
  if (s3Client == null) {
    s3Client = new S3Client()
  }
```